src/mlmd/LineageView.tsx

```diff
@@ -51,10 +51,12 @@ export interface LineageViewProps {
   target: Artifact;
   cardWidth?: number;
   columnPadding?: number;
-  buildResourceDetailsPageRoute(resource: LineageResource, typeName: string): string
+  buildResourceDetailsPageRoute(resource: LineageResource, typeName: string): string;
 }
 
 interface LineageViewState {
+  // Dynamic value set to 1/5th of the view width.
+  columnWidth: number;
```

Comment:
The goal of this is to allow the canvas, columns, and lines be passed props instead of relying on measuring the DOM structure of the siblings and parents, which is brittle and broke once i started introducing changes. 

Happy to make changes to achieve what you're describing if you can be more specific, but I  don't want to re-introduce changes where a component is measuring its parents and siblings.

Reply:
I think the way it is currently is best in that case. Let's go with this approach!